{black_filter_enabled: true, black_hue: 0.0, black_hue_variance: 0.0, black_hue_weight: 0.0,
  black_sat: 124.94999999999999, black_sat_variance: 800.0, black_sat_weight: 1.0,
  black_threshold: 0.29999999999999999, black_val: 102.0, black_val_variance: 800.0,
  black_val_weight: 0.32000000000000001, blue_filter_enabled: true, blue_hue: 104.40000000000001,
  blue_hue_variance: 200.0, blue_hue_weight: 1.0, blue_sat: 198.89999999999998, blue_sat_variance: 200.0,
  blue_sat_weight: 0.5, blue_threshold: 0.20000000000000001, blue_val: 0.0, blue_val_variance: 200.0,
  blue_val_weight: 0.5, enable_meanshift: true, enable_thresholding: false, green_filter_enabled: true,
  green_hue: 84.569999999999993, green_hue_variance: 13.140000000000001, green_hue_weight: 1.0,
  green_sat: 174.0, green_sat_variance: 111.3, green_sat_weight: 0.10000000000000001,
  green_threshold: 0.20000000000000001, green_val: 236.62, green_val_variance: 815.34799999999996,
  green_val_weight: 0.10000000000000001, ms_color_ratius: 1.0, ms_max_iter: 2, ms_max_level: 2,
  ms_min_epsilon: 1.0, ms_spatial_radius: 1.0, orange_filter_enabled: true, orange_hue: 40.847999999999999,
  orange_hue_variance: 1211.268, orange_hue_weight: 1.0, orange_sat: 18.997, orange_sat_variance: 360.60000000000002,
  orange_sat_weight: 0.10000000000000001, orange_threshold: 0.20000000000000001, orange_val: 235.541,
  orange_val_variance: 1094.8499999999999, orange_val_weight: 0.10000000000000001,
  red_filter_enabled: true, red_hue: 3.6000000000000001, red_hue_variance: 500.0,
  red_hue_weight: 1.0, red_sat: 25.5, red_sat_variance: 1000.0, red_sat_weight: 0.40000000000000002,
  red_threshold: 0.29999999999999999, red_val: 76.5, red_val_variance: 1000.0, red_val_weight: 0.40000000000000002,
  threshold_max_value: 255.0, threshold_type: 1, threshold_value: 127.0, white_filter_enabled: true,
  white_hue: 0.0, white_hue_variance: 0.0, white_hue_weight: 0.0, white_sat: 162.0,
  white_sat_variance: 200.0, white_sat_weight: 0.25, white_threshold: 0.20000000000000001,
  white_val: 254.99999999999997, white_val_variance: 200.0, white_val_weight: 0.5,
  yellow_filter_enabled: false, yellow_hue: 30.0, yellow_hue_variance: 0.0, yellow_hue_weight: 0.0,
  yellow_sat: 255.0, yellow_sat_variance: 0.0, yellow_sat_weight: 0.0, yellow_threshold: 0.0,
  yellow_val: 255.0, yellow_val_variance: 0.0, yellow_val_weight: 0.0}
